.github/workflows/fpga.yml

```diff
@@ -249,7 +249,11 @@ jobs:
                        package(caliptra-builder) |
                        package(caliptra-cfi-derive) |
                        package(caliptra-file-header-fix) |
-                       package(compliance-test))'
+                       package(compliance-test) |
+                       package(caliptra-drivers) |
+                       package(caliptra-runtime) |
```

Comment:
I think it would be good to filter out the 18 failing tests, so we can keep most of the runtime test suite